test/.eslintrc.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "eslint:recommended",
+  "parserOptions": {
+    "sourceType": "module",
+    "ecmaVersion": 8
+  },
+  "env": {
+    "es6": true,
```

Comment:
This needs to be at least version 11 for the `import.meta.url` to work. 

I think we need to decide if we want to go with es6 or jump all the way up to es2020 (which version 11 is part of). It is possible to keep `env es6` and use `ecmaVersion 11`, but not sure if that is the way to go here. (I'd rather just jump all the way up to es2020, or es2021) 

[Ref this discussion here](https://stackoverflow.com/a/65541635)

Reply:
I just need to find a way to load the json file that makes everyone happy (ie tests and eslint)

Reply:
Yes, let’s not use require here. Let’s use fs.readFileSync or similar.

Reply:
You can also rename the file to a `.js` instead of `.json`, and inside of the js-file you can do a `default export [...]`. This way its possible to import the js-file into the tests and they will run as usual. Then you wont need fs.readFileSync. 

Not sure what you usual coding practices have been with d3js, so I'll let you guys decide whats best :)

Reply:
now let's see if it passes CI :)